lib/grover/js/processor.js

```diff
@@ -190,6 +202,23 @@ const _processPage = (async (convertAction, urlOrHtml, options) => {
       await page.hover(hoverSelector);
     }
 
+    if (errors.length > 0) {
+      function RequestFailedError(errors = []) {
```

Comment:
do you need a default here given we know there are errors from the line above?

Reply:
No, just being over-defensive :-P Will remove.
```suggestion
      function RequestFailedError(errors) {
```